<!DOCTYPE  html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><title>Branches of Bioinformatics</title><link href="navigation.css" rel="stylesheet" type="text/css"/><link href="document.css" rel="stylesheet" type="text/css"/></head><body><p class="top_nav"><a href="part611.htm">&lt; Назад</a><span> | </span><a href="../index.html">Содержимое</a><span> | </span><a href="part613.htm">Далее &gt;</a></p><p style="padding-left: 6pt;text-indent: 0pt;text-align: left;"><a href="#bookmark610" name="bookmark2687">Branches of Bioinformatics</a><a name="bookmark2708">&zwnj;</a></p><p style="padding-top: 5pt;padding-left: 6pt;text-indent: 0pt;text-align: justify;">Some of the main branches of bioinformatics are Genomics, Proteomics, Computer-Aided Drug Design, Data Mining of Bio Databases, Molecular Phylogenetics, Microarray Data Analysis, and Molecular Modeling of Biomolecules. They are described here.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-left: 6pt;text-indent: 0pt;text-align: left;">Genomics</h4><p style="padding-top: 5pt;padding-left: 6pt;text-indent: 0pt;text-align: justify;">The area of <i>Genomics </i>deals with the following:</p><p style="padding-top: 9pt;padding-left: 38pt;text-indent: -15pt;text-align: justify;">1. <b>Gene finding. </b>It deals with finding the genes for a given DNA sequence. Gene finders are useful for identifying coding regions from which one can deduce the structure of genes and resulting proteins. <i>HMMgene </i>and <i>Genscan </i>are widely used gene finders for eukaryotic genome, and <i>Glimmer </i>is a widely used gene finder for microbial gene prediction.</p><p style="padding-top: 4pt;padding-left: 38pt;text-indent: -15pt;text-align: justify;">2. <b>Genome sequence alignment analysis. </b>It deals with finding how similar are a set of two or more DNA sequences. Pair-wise sequence alignment analysis deals with finding how similar are two DNA sequences, whereas multiple sequence alignment analysis deals with analyzing how similar are a set of more than two DNA sequences. Genome sequence analysis involves searching of sequence databases using analysis tools for revealing sequence  similarities  that  can  throw  light  into  their  structural  and  functional characteristics. Various analysis tools are available for genome sequence analysis. Tools based on dynamic programming methods (like <i>Smith-Waterman</i>) are popular due to their ability to provide an optimal alignment, while those based on heuristics methods (like <i>BLAST </i>and <i>FASTA</i>) are widely used due to their ability to search databases faster.</p><p style="padding-top: 4pt;padding-left: 38pt;text-indent: -15pt;text-align: justify;">3. <b>Genome browsing. </b>It deals with finding how to visualize a given DNA sequence insightfully.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-left: 5pt;text-indent: 0pt;text-align: left;">Proteomics</h4><p style="padding-top: 5pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">The area of <i>Proteomics </i>deals with the following:</p><p style="padding-top: 9pt;padding-left: 23pt;text-indent: 0pt;text-align: left;">1. <b>Protein sequence alignment analysis. </b>It deals with analyzing how similar are a set of</p><p style="padding-top: 3pt;padding-left: 39pt;text-indent: 0pt;text-align: justify;"><a name="bookmark2709">two or more protein sequences. Pair-wise sequence alignment analysis deals with finding how similar is a protein sequences data with another one, whereas multiple sequence alignment analysis deals with analyzing how similar are a set of more than two protein sequences. Sequence alignment technique is widely applicable to both genomics and proteomics and both make use of the same analysis tools.</a></p><p style="padding-top: 4pt;padding-left: 39pt;text-indent: -15pt;text-align: left;">2. <b>Protein folding problem. </b>It deals with finding the primary, secondary, or tertiary structure of a given molecule.</p><p style="padding-top: 4pt;padding-left: 39pt;text-indent: -15pt;text-align: left;">3. <b>Active site determination problem. </b>It deals with finding which part of a body is chemically most active.</p><p style="padding-top: 4pt;padding-left: 39pt;text-indent: -15pt;text-align: left;">4. <b>Protein-protein interaction problem. </b>It deals with finding how would one protein interact with another protein.</p><p style="padding-top: 4pt;padding-left: 39pt;text-indent: -15pt;text-align: left;">5. <b>Protein sorting problem. </b>It deals with finding to which cell compartment a given protein belongs.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-left: 6pt;text-indent: 0pt;text-align: left;">Computer-Aided Drug Design</h4><p style="padding-top: 5pt;padding-left: 6pt;text-indent: 0pt;text-align: justify;">The area of <i>computer-aided drug design </i>deals with using computational techniques to minimize time and effort for searching drug molecules. Identifying a disease and bringing out an effective drug for it takes about 10-15 years, costs enormously, and involves testing of thousands of candidate molecules. Traditionally, drug discovery was based on blind search for molecules, which was costly and time-consuming. Computer-aided drug design techniques have helped in substantially reducing the drug discovery cost and time. These techniques help in performing computer-aided analysis of biomolecular interactions, which are core to all regulatory and metabolic processes that together constitute the process of life. Automated analysis and prediction of biomolecular interactions is useful in rational drug design process. Biomolecular Docking software, like <i>FTDock </i>and <i>DARWIN, </i>are widely used for this purpose.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-left: 6pt;text-indent: 0pt;text-align: left;">Data Mining of Bio Databases</h4><p class="s16" style="padding-top: 5pt;padding-left: 6pt;text-indent: 0pt;text-align: justify;">Bio databases <span class="p">are huge databases of mostly sequence data. Numerous genome sequencing projects across the world, like Human Genome Project, have led to millions of sequence data pouring into bio databases. Among several available bio databases, the popular ones are European Molecular Biology Laboratory DNA database (EMBL), GenBank at National Centre for Biotechnology information, Bethesda and DNA Data Bank (DDB), Japan, and SWISS- PROT, a protein sequence database at Swiss Institute of Bioinformatics, Geneva. Researchers often use data mining techniques and other computational tools to analyze the data in bio databases for extracting useful information, such as structural and functional information of an unknown gene or protein.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-left: 6pt;text-indent: 0pt;text-align: left;">Molecular Phylogenetics</h4><p style="padding-top: 5pt;padding-left: 6pt;text-indent: 0pt;text-align: justify;">The area of <i>molecular phylogentics </i>deals with using computers to make classes, super classes, and abstract classes of all organisms. A phylogenetic tree is a pictorial representation of such classifications. It helps in the study of evolution of living beings. Use of computers has helped biologists in creating complex phylogenetic trees based on quantitative analysis of DNA/protein</p><p style="padding-top: 3pt;padding-left: 6pt;text-indent: 0pt;text-align: justify;">sequences for preparing all classifications. Most phylogentic trees are computed based on multiple-sequence alignment of protein sequences of the set of organisms we want to classify. This often requires quite involved computation, including facing some intractable problems.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-left: 6pt;text-indent: 0pt;text-align: justify;">Microarray Data Analysis</h4><p style="padding-top: 5pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;"><a name="bookmark2710">Microarrays are tiny chips used to study a phenomenon called </a><i>gene expression</i>. Not all genes in all cells are active at all times. Which of them are “expressed” at any given time or situation is the question that microarrays help to answer. <i>Microarray data analysis </i>deals with the use of computational methods for monitoring and analyzing gene expression profiles of thousands of genes simultaneously. This has great significance in several areas, such as novel gene identification, disease diagnosis, and drug discovery. Researchers have developed and applied several computational algorithms to uncover co-regulated genes in microarrays. The co- regulation of genes could probably be due to the presence of similar cis-regulatory elements in upstream regions of gene sequences. Tools for microarray data analysis help to identify such significant elements in upstream regions. <i>MEME </i>(Multiple Expectation Maximization for motif Elicitation) is one such tool for detecting motifs in a set of DNA or protein sequences.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><h4 style="padding-left: 5pt;text-indent: 0pt;text-align: justify;">Molecular Modeling of Biomolecules</h4><p style="padding-top: 5pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">Detailed structure and dynamics of many biomolecular systems at molecular level are difficult to discern by conventional methods such as X-ray, NMR spectroscopy, etc. Researchers often use Molecular Dynamics (MD) simulation, a deterministic molecular modeling method, as a complementary method for providing a great deal of information on structure and dynamics of the biomolecule under investigation. MD simulation derives sets of atomic positions in sequence by solving the differential equations embodied in Newton&#39;s law of motion. Molecular modeling programs like <i>AMBER </i>and <i>CHARMM </i>are widely used to carry out MD simulations of biomolecules.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="nav">&nbsp;&nbsp;</p><p class="nav">&nbsp;</p><p class="nav"><a href="part611.htm">&lt; Назад</a><span> | </span><a href="../index.html">Содержимое</a><span> | </span><a href="part613.htm">Далее &gt;</a></p><p class="nav">&nbsp;&nbsp;</p></body></html>
